cypress/e2e/tests/pages/fleet/dashboard.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { FleetDashboardPagePo } from '@/cypress/e2e/po/pages/fleet/fleet-dashboard.po';
+import { GitRepoCreatePo } from '@/cypress/e2e/po/pages/fleet/gitrepo-create.po';
+
+describe.only('Fleet Dashboard', { tags: '@adminUser' }, () => {
+  let fleetDashboardPage: FleetDashboardPagePo;
+  const repoName = 'fleet-e2e-test-dashboard';
+
+  beforeEach(() => {
+    cy.login();
+    fleetDashboardPage = new FleetDashboardPagePo('local');
+    fleetDashboardPage.goTo();
+  });
+
+  beforeEach(() => { });
+
+  before(() => {
+    cy.login();
+
+    const gitRepoCreatePage = new GitRepoCreatePo('local');
+
+    gitRepoCreatePage.goTo();
+    gitRepoCreatePage.setRepoName(repoName);
+    gitRepoCreatePage.selectWorkspace('fleet-local');
+    gitRepoCreatePage.setGitRepoUrl('https://github.com/Shavindra/fleet-examples.git');
+    gitRepoCreatePage.setBranchName();
+    // NB - This step is here because DOM may not be ready
+    gitRepoCreatePage.goToNext();
+    gitRepoCreatePage.create();
+  });
+
+  it('Should display cluster status', () => {
+    const fleetLocalResourceTable = fleetDashboardPage.resourceTable('fleet-local');
+    const row = fleetLocalResourceTable.sortableTable().rowElements().eq(0);
```

Comment:
The rows (plural 🤓) should be handled within the POs, either fleet or extending the table.